sunspot/lib/sunspot/field_factory.rb

```diff
@@ -132,7 +132,7 @@ class Dynamic < Abstract
       def initialize(name, type, options = {}, &block)
         super(name, options, &block)
         @type, @options = type, options
-        @separator = @options.delete(:separator) || ':'
```

Comment:
Why the change to the separator?

Reply:
#229 pointed me to it: https://lucene.apache.org/solr/guide/7_0/defining-fields.html
> Field names should consist of alphanumeric or underscore characters only and not start with a digit. This is not currently strictly enforced, but other field names will not have first class support from all components and back compatibility is not guaranteed.

Reply:
👍